packages/polar/src/types.ts

```diff
@@ -13,7 +13,7 @@
 // fields), we don't use `extends` as that can interfere with plugin authors
 // trying to augment the config types.
 // Networks config\
-import { FeePool } from "secretjs/dist/protobuf_stuff/cosmos/distribution/v1beta1/distribution";
+import { FeePool } from "secretjs/dist/protobuf/cosmos/distribution/v1beta1/distribution";
```

Comment:
Can we copy paster FeePool type here ? 
This code may break because we are not accessing FeePool using index.js export